mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -178,6 +178,7 @@ private[recommendation] trait ALSParams extends ALSModelParams with HasMaxIter w
  * @param itemFactors a DataFrame that stores item factors in two columns: `id` and `features`
  */
 @Experimental
+@Since("1.3.0")
 class ALSModel private[ml] (
     override val uid: String,
```

Comment:
document public params in the constructor

~~~scala
    @Since("1.4.0") override val uid: String,
    @Since("1.4.0") override val rank: Int
~~~